README.md

```diff
@@ -356,6 +357,7 @@ CI-Tests                    | Does the project run tests in CI, e.g. [GitHub Act
 CII-Best-Practices          | Does the project have a [CII Best Practices Badge](https://bestpractices.coreinfrastructure.org/en)?
 Code-Review                 | Does the project require code review before code is merged?
 Contributors                | Does the project have contributors from at least two different organizations?
+Dangerous-Workflow          | Does the project have GitHub Action workflows avoid dangerous patterns?
```

Comment:
nit:
"Does the project have GitHub Action workflows **that** avoid dangerous patterns?"